Big refactor

LoveRenamon opened the pull request:
Also changes both eye sphere primitive to be created already rotated X90; It make easier to find out the _"Angle of deviantion from center"_ angle by simply rotate the Z-Axis and use such euler angle angle as source do;

 I would like to know if is better make the eyelid opational since HWM don't necessary need then

LoveRenamon committed:
add DmxEyelid for Qc eyes   change sphere primitive to be created already rotated X90°. It make easier to find out the "Angle of deviantion from center" angle by simply rotate the Z-Axis

LoveRenamon committed:
Big big refactor here.
Now this thing is compatible with Blender 2.83+, also in 3.1+ and 3.5+
 there's a bund of experimental stuff and its almost nothing like the Progonal REDs-Tools
	Annnd because that I'm renaming this to 'Source Engine Tools' which is more widen name and well suited to.

This addon now works optionally along:
	- CATs
	- Blender Source Tools
	- SourceOps

And planned to get well with:
	-  Hagget's Tools
	-  SourceIO

To expand it capabilities.

I need to make a README but this is not even close to a priority over here...

 Changes to be committed:
	modified:   __init__.py
	new file:   batch_smd_export.py
	new file:   bone_lists/bones.py
	new file:   bone_lists/bones_aponia.py
	new file:   bone_lists/bones_attachments.py
	new file:   bone_lists/bones_bad.py
	new file:   bone_lists/bones_breast.py
	new file:   bone_lists/bones_facials.py
	new file:   bone_lists/bones_helpers.py
	new file:   bone_lists/bones_mmdbase.py
	new file:   bone_lists/bones_more.py
	new file:   bone_lists/bones_tx.py
	new file:   bone_lists/bones_valve.py
	new file:   bone_lists/bones_without_deform.py
	new file:   clear_blank_shape_keys.py
	new file:   copy_pose.py
	new file:   copy_procedural.py
	new file:   create_facs.py
	new file:   empty_to_bones.py
	modified:   mesh_operators.py
	modified:   operators.py
	new file:   qc_eyes.py
	new file:   rename_dicts/bio3_to_tf2.py
	new file:   rename_dicts/blender_to_valvebiped.py
	new file:   rename_dicts/cybersleuth_to_blender.py
	new file:   rename_dicts/fortnite_to_blender.py
	new file:   rename_dicts/kofas_blender.py
	new file:   rename_dicts/mmd_to_valvebiped.py
	new file:   rename_dicts/rigfy_to_blender.py
	new file:   rename_dicts/rxd_blender.py
	new file:   rename_dicts/tf2_to_blender.py
	new file:   rename_dicts/valvebiped_to_blender.py
	new file:   selected_bones.py
	new file:   setup_armature.py
	modified:   texture_tools.py
	modified:   tools_panel.py
	deleted:    rename_bones.py

REDxEYE commented:
Holly molly!

REDxEYE commented on __init__.py:
```diff
@@ -1,157 +1,168 @@
+    "location": "View3D > UI > Tool",
+    "description": "Multi-purpose tools for preparing armature for Source Engine export, derivated from the deprecated RED Tool's",
+    'warning': '',
+    'wiki_url': 'https://github.com/LoveRenamon/REDs-Tools/wiki',
```
Update url to original repo before merge

LoveRenamon replied:
eh?
I even forgot I did this pull request haha
let this as draft for now, I'll do a bit more changes in a few weeks  which includes the right wiki url from your repo

REDxEYE replied:
Okay. Can you also keep the original functionality? You removed a lot of utility tools from it

REDxEYE commented:
Can you translate comments to English?

LoveRenamon commented:
sure, I'll translate them

LoveRenamon committed:
change wiki and addon name (again)

LoveRenamon committed:
portuguese to english comments

LoveRenamon committed:
Fix sily issue with qc_eyes.py

 Changes to be committed:
	modified:   __init__.py
	modified:   qc_eyes.py

LoveRenamon committed:
finetune procedurals in operators.py

LoveRenamon committed:
disable procedurals painel

LoveRenamon committed:
add gitignore

LoveRenamon committed:
fix unregister EyeFormats missing "s"

LoveRenamon committed:
Work in progress a method to rename active object Vertex Groups without rely on armature

REDxEYE committed:
merge 0cbe95526f3bd064484e8cf77faaace8c340f41c
Finally moved tools to left panel
Added more shape key tools

REDxEYE committed:
Typo fix

LoveRenamon committed:
moar notes for FACs

LoveRenamon committed:
operators to enfor VertexGroups and support renaming bones in Blender 2.8x

LoveRenamon committed:
update CyberSleuth dictionary

REDxEYE commented on __init__.py:
```diff
@@ -19,18 +19,18 @@
+    "description": "Multi-purpose tools preparing armature for Source Engine export",
     'warning': '',
-    'wiki_url': 'https://github.com/LoveRenamon/REDs-Tools/wiki',
+    'wiki_url': 'https://github.com/rEDxEYE/REDs-Tools/wiki',
```
Typo here

LoveRenamon committed:
create_facs.py ensure creates only necessary Shape Keys

LoveRenamon committed:
copy_pose.py ensure behaviours
	now works only with armatures, so theres no exceptions
	add support to multiple armature be parsed at once

LoveRenamon committed:
fix clear_blank_shape_keys operator

LoveRenamon committed:
fix wiki link

LoveRenamon committed:
chages create_facs.py
	translate to english
	allow add Shapes to multiple objects at once

LoveRenamon committed:
aid VALVE_OT_MergeBoneWeights behaviour
	ensure this options enables only at the very specific case to work AND tell to user what's wrong

althrough VALVE_OT_ConnectBones works, it still connects bones to the wrong rolls..

LoveRenamon committed:
reenable VALVE_PT_TextureTools

LoveRenamon committed:
change VALVE_PT back to VIEW3D_PT to avoid naming confusion at my fork

LoveRenamon committed:
REDxEYE got still working at this alpha channel separator, so even if trivial, let it enabled

LoveRenamon committed:
this got to be replaced by another module.
Also too broken to be used as for now

LoveRenamon committed:
fix sily condition

LoveRenamon committed:
fix "Transferer shapes" not working under inexistent baseline Shape Key
	also add some failsafe conditions in usage
